dotnet/src/Azure.Iot.Operations.Services/AssetAndDeviceRegistry/Models/CreateDiscoveredAssetEndpointProfileRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.Operations.Services.AssetAndDeviceRegistry.Models;
+
+public record CreateDiscoveredAssetEndpointProfileRequest
+{
+    public string? AdditionalConfiguration { get; set; } = default;
+
+    public string? DaepName { get; set; } = default;
```

Comment:
This name is also a bit clunky. Maybe the DTDL could just call this field "name" since the "discovered asset endpoint profile" portion of the name is covered by the class name?